test/docker-e2e/e2e_minor_version_compatibility_test.go

```diff
@@ -41,9 +41,10 @@ func (s *CelestiaTestSuite) TestMinorVersionCompatibility() {
 		name string
 		tags []string
 	}{
+
 		{
-			name: "v5 minor versions",
```

Comment:
v5 was removed as the test is failing on most of `v5.x` and this is an expected behaviour, instead a number of v6.x minors are added

Reply:
If v5.x was removed then the logs in the PR description seem out of date:

```
...
=== NAME  TestCelestiaTestSuite
    setup.go:223: Pruned unused networks: [tastora-qytshnuf]
--- FAIL: TestCelestiaTestSuite (209.12s)
    --- FAIL: TestCelestiaTestSuite/TestMinorVersionCompatibility (208.85s)
        --- FAIL: TestCelestiaTestSuite/TestMinorVersionCompatibility/v5_minor_versions (63.84s)
        --- PASS: TestCelestiaTestSuite/TestMinorVersionCompatibility/v6_minor_versions (145.00s)
...
```